2021-04-26 07:26:26,071 INFO    MainThread:6872 [wandb_setup.py:_flush():69] setting env: {}
2021-04-26 07:26:26,073 INFO    MainThread:6872 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-26 07:26:26,073 INFO    MainThread:6872 [wandb_init.py:_log_setup():336] Logging user logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210426_072626-3olnb2k5/logs/debug.log
2021-04-26 07:26:26,074 INFO    MainThread:6872 [wandb_init.py:_log_setup():337] Logging internal logs to /n/holyscratch01/Academic-cluster/Spring_2021/g_84102/SCRATCH/ImageNet/wandb/run-20210426_072626-3olnb2k5/logs/debug-internal.log
2021-04-26 07:26:26,074 INFO    MainThread:6872 [wandb_init.py:init():369] calling init triggers
2021-04-26 07:26:26,074 INFO    MainThread:6872 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {}
2021-04-26 07:26:26,074 INFO    MainThread:6872 [wandb_init.py:init():418] starting backend
2021-04-26 07:26:26,075 INFO    MainThread:6872 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-26 07:26:26,148 INFO    MainThread:6872 [backend.py:ensure_launched():123] starting backend process...
2021-04-26 07:26:26,202 INFO    MainThread:6872 [backend.py:ensure_launched():127] started backend process with pid: 6958
2021-04-26 07:26:26,205 INFO    MainThread:6872 [wandb_init.py:init():423] backend started and connected
2021-04-26 07:26:26,209 INFO    MainThread:6872 [wandb_init.py:init():465] updated telemetry
2021-04-26 07:26:26,211 INFO    MainThread:6872 [wandb_init.py:init():484] communicating current version
2021-04-26 07:26:27,835 INFO    MainThread:6872 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-26 07:26:27,836 INFO    MainThread:6872 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-26 07:26:28,004 INFO    MainThread:6872 [wandb_init.py:init():522] starting run threads in backend
2021-04-26 07:26:31,363 INFO    MainThread:6872 [wandb_run.py:_console_start():1524] atexit reg
2021-04-26 07:26:31,366 INFO    MainThread:6872 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-26 07:26:31,367 INFO    MainThread:6872 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-26 07:26:31,369 INFO    MainThread:6872 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-26 07:26:31,370 INFO    MainThread:6872 [wandb_init.py:init():546] run started, returning control to user process
2021-04-26 07:26:31,372 INFO    MainThread:6872 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-26 07:26:31,373 INFO    MainThread:6872 [wandb_run.py:_restore():1466] restore
2021-04-26 07:26:33,518 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 547
  total_bytes: 547
}

2021-04-26 07:26:33,672 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 547
  total_bytes: 547
}

2021-04-26 07:26:34,067 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 547
  total_bytes: 6042
}

2021-04-26 07:26:34,216 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 547
  total_bytes: 6042
}

2021-04-26 07:26:34,319 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6042
  total_bytes: 6042
}

2021-04-26 07:26:34,421 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6042
  total_bytes: 6042
}

2021-04-26 07:26:34,524 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6042
  total_bytes: 6042
}

2021-04-26 07:26:34,627 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6042
  total_bytes: 6042
}

2021-04-26 07:26:34,729 INFO    MainThread:6872 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6042
  total_bytes: 6042
}

2021-04-26 07:26:36,136 INFO    MainThread:6872 [wandb_run.py:_show_files():1838] logging synced files
